src/modules/IndexContentRenderer.ts

```diff
@@ -40,8 +40,8 @@ export class IndexContentRenderer extends MarkdownRenderChild {
 				const headings = this.app.metadataCache.getFileCache(value).headings
 				const fileLink = this.app.metadataCache.fileToLinktext(value, this.filePath)
 
-				list.push(`1. [[${fileLink}|${value.basename}]]`)
-				if (headings != null && !this.plugin.settings.disableHeadlines) {
+				list.push(`1. ${this.plugin.settings.includeFileContent? '!': ''}[[${fileLink}|${value.basename}]]`);
+        if (headings != null && !this.plugin.settings.disableHeadlines) {
```

Comment:
There is an indent missing, please add this :)

Reply:
Yuck. 
Indent inserted..
Thanks